### ISSUE TITLE
IllegalArgumentException on 1.17

### ISSUE THREAD

**Issue by SlimeDog** [Niall7459/HolographicExtension#44]
Spigot 1.17 or Paper 1.17
ProtocolLib 4.7.0-SNAPSHOT-b522
PlaceholderAPI version 2.10.10-DEV-123
HolographicExtension version 1.10.9

Update needed to play nicely with the above configuration.
```
[15:51:44] [Server thread/ERROR]: [HolographicExtension] Unhandled exception occured in onPacketSending(PacketEvent) for HolographicExtension
java.lang.RuntimeException: Unable to clone object PacketContainer[type=ENTITY_METADATA[class=PacketPlayOutEntityMetadata, id=77], structureModifier=StructureModifier[fieldType=class java.lang.Object, data=[private final int net.minecraft.network.protocol.game.PacketPlayOutEntityMetadata.a, private final java.util.List net.minecraft.network.protocol.game.PacketPlayOutEntityMetadata.b]]] (com.comphenix.protocol.events.PacketContainer)
        at com.comphenix.protocol.reflect.cloning.SerializableCloner.clone(SerializableCloner.java:44) ~[?:?]
        at com.comphenix.protocol.events.PacketContainer.deepClone(PacketContainer.java:249) ~[?:?]
        at net.kitesoftware.holograms.listener.PacketPlaceholderListener.onPacketSending(PacketPlaceholderListener.java:56) ~[?:?]
        at com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:195) ~[?:?]
        at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149) ~[?:?]
        at com.comphenix.protocol.injector.PacketFilterManager.handlePacket(PacketFilterManager.java:535) ~[?:?]
        at com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:516) ~[?:?]
        at com.comphenix.protocol.injector.netty.ProtocolInjector.packetQueued(ProtocolInjector.java:346) ~[?:?]
        at com.comphenix.protocol.injector.netty.ProtocolInjector.onPacketSending(ProtocolInjector.java:308) ~[?:?]
        at com.comphenix.protocol.injector.netty.ChannelInjector.processSending(ChannelInjector.java:433) ~[?:?]
        at com.comphenix.protocol.injector.netty.ChannelInjector.access$800(ChannelInjector.java:70) ~[?:?]
        at com.comphenix.protocol.injector.netty.ChannelInjector$3.handleScheduled(ChannelInjector.java:373) ~[?:?]
        at com.comphenix.protocol.injector.netty.ChannelInjector$3.onMessageScheduled(ChannelInjector.java:343) ~[?:?]
        at com.comphenix.protocol.injector.netty.ChannelProxy$2.schedulingRunnable(ChannelProxy.java:127) ~[?:?]
        at com.comphenix.protocol.injector.netty.EventLoopProxy.execute(EventLoopProxy.java:95) ~[?:?]
        at net.minecraft.network.Connection.sendPacket(Connection.java:294) ~[?:?]
        at net.minecraft.network.Connection.send(Connection.java:258) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:1920) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:1906) ~[?:?]
        at net.minecraft.server.level.ServerEntity.a(ServerEntity.java:285) ~[?:?]
        at net.minecraft.server.level.ServerEntity.addPairing(ServerEntity.java:268) ~[?:?]
        at net.minecraft.server.level.ChunkMap$TrackedEntity.updatePlayer(ChunkMap.java:2381) ~[?:?]
        at net.minecraft.server.level.ChunkMap$TrackedEntity.updatePlayers(ChunkMap.java:2292) ~[?:?]
        at net.minecraft.server.level.ChunkMap.addEntity(ChunkMap.java:1991) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.level.ServerChunkCache.addEntity(ServerChunkCache.java:950) ~[?:?]
        at net.minecraft.server.level.ServerLevel$EntityCallbacks.onTrackingStart(ServerLevel.java:2179) ~[?:?]
        at net.minecraft.server.level.ServerLevel$EntityCallbacks.onTrackingStart(ServerLevel.java:2137) ~[?:?]
        at net.minecraft.world.level.entity.PersistentEntitySectionManager.startTracking(PersistentEntitySectionManager.java:140) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.world.level.entity.PersistentEntitySectionManager.addEntity(PersistentEntitySectionManager.java:103) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.world.level.entity.PersistentEntitySectionManager.addNewEntity(PersistentEntitySectionManager.java:86) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.level.ServerLevel.addEntity0(ServerLevel.java:1221) ~[?:?]
        at net.minecraft.server.level.ServerLevel.addEntity(ServerLevel.java:1128) ~[?:?]
        at org.bukkit.craftbukkit.v1_17_R1.CraftWorld.dropItem(CraftWorld.java:774) ~[patched_1.17.jar:git-Paper-49]
        at org.bukkit.craftbukkit.v1_17_R1.CraftWorld.dropItem(CraftWorld.java:763) ~[patched_1.17.jar:git-Paper-49]
        at one.lindegaard.MobHunting.rewards.RewardManager.dropMoneyOnGround_RewardManager(RewardManager.java:350) ~[?:?]
        at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1720) ~[?:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.17.jar:git-Paper-49]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.17.jar:git-Paper-49]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.17.jar:git-Paper-49]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.17.jar:git-Paper-49]
        at org.bukkit.craftbukkit.v1_17_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:843) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.world.entity.LivingEntity.dropAllDeathLoot(LivingEntity.java:1703) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.die(LivingEntity.java:1624) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.hurt(LivingEntity.java:1455) ~[?:?]
        at net.minecraft.world.entity.projectile.AbstractArrow.onHitEntity(AbstractArrow.java:397) ~[?:?]
        at net.minecraft.world.entity.projectile.Projectile.onHit(Projectile.java:181) ~[?:?]
        at net.minecraft.world.entity.projectile.Projectile.preOnHit(Projectile.java:172) ~[?:?]
        at net.minecraft.world.entity.projectile.AbstractArrow.tick(AbstractArrow.java:241) ~[?:?]
        at net.minecraft.world.entity.projectile.Arrow.tick(Arrow.java:112) ~[?:?]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:942) ~[?:?]
        at net.minecraft.world.level.Level.guardEntityTick(Level.java:857) ~[?:?]
        at net.minecraft.server.level.ServerLevel.lambda$tick$5(ServerLevel.java:639) ~[?:?]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:54) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:620) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1551) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:477) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1404) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1180) ~[patched_1.17.jar:git-Paper-49]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[patched_1.17.jar:git-Paper-49]
        at java.lang.Thread.run(Thread.java:831) [?:?]
        at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.IllegalArgumentException: Unable to find a method that matches {modifiers=[required: 1000, banned: 0], return=Exact class net.minecraft.network.protocol.game.PacketPlayOutEntityMetadata, params=[{Type: Exact class net.minecraft.network.PacketDataSerializer, Index: 0}], paramCount=1}
        at com.comphenix.protocol.reflect.FuzzyReflection.getMethod(FuzzyReflection.java:174) ~[?:?]
        at com.comphenix.protocol.events.PacketContainer.readObject(PacketContainer.java:323) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor230.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:567) ~[?:?]
        at java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1224) ~[?:?]
        at java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2378) ~[?:?]
        at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2212) ~[?:?]
        at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1689) ~[?:?]
        at java.io.ObjectInputStream.readObject(ObjectInputStream.java:495) ~[?:?]
        at java.io.ObjectInputStream.readObject(ObjectInputStream.java:453) ~[?:?]
        at com.comphenix.protocol.reflect.cloning.SerializableCloner.clone(SerializableCloner.java:42) ~[?:?]
        ... 59 more
```

**Comment by thiagogebrimm** [comment]
I have the same problem, I hope they update soon :(

**Comment by ProfessorZel** [comment]
Also encounter same problem.

**Comment by SlimeDog** [comment]
Fixed by updated to HolographicDisplays 2.4.9.
